.github/workflows/maven.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     - name: Set up JDK
       uses: actions/setup-java@v4
       with:
-        java-version: 17
+        java-version: 21
```

Comment:
I think we shouldn't change the version here, without moving the whole project to 21. As said I would tackle that in a different PR

Reply:
ah ok, sorry, will revert